### DISCUSSION TITLE
[Feature] 8卡A100*80G评测Qwen1.5-72B推荐的参数设置是怎么样的呢

### DISCUSSION THREAD

**Discussion by boundles** [open-compass/opencompass#1138] [Q&A]
### Describe the feature

现在采用的参数如下：
`models = [
    dict(
        type=HuggingFaceCausalLM,
        abbr='qwen1.5-72b-chat-hf',
        path="Qwen1.5-72B-Chat",
        model_kwargs=dict(
            device_map='auto',
            trust_remote_code=True
        ),
        tokenizer_kwargs=dict(
            padding_side='left',
            truncation_side='left',
            trust_remote_code=True,
            use_fast=False,
        ),
        meta_template=_meta_template,
        max_out_len=100,
        max_seq_len=2048,
        batch_size=8,
        run_cfg=dict(num_gpus=4, num_procs=1),
        end_str='<|im_end|>',
        batch_padding=True,
    )
]`

感觉评测很慢，20多个小时都没出结果

### Will you implement it?

- [ ] I would like to implement this feature and create a PR!

**Comment by disperaller** [comment]
+1
